tests/test_viz.py

```diff
@@ -87,6 +87,34 @@ def test_plot_metadata_facet_by_boxplot(ocx, api_data):
     assert chart.encoding.x.axis.title == ""
 
 
+def test_plot_metadata_secondary_haxis_scatter(ocx, api_data):
+    samples = ocx.Samples.where(project="4b53797444f846c4")
```

Comment:
Nit: looks like this project could be a fixture vs a hardcoded UUID?

Reply:
Ah, the data is composed of API mocks, nevermind!

Reply:
Good idea, I'll make a fixture for the SampleCollection